### PR TITLE
Krbundy validation

### PR DESCRIPTION
**Pull request by krbundy**
Merging validation updates into the development branch. This includes:

- "translation" of Python code into R, including aggregation, figures, and data pre-processing.
- additional model training scripts and training data pre-processing
- Addition of normalization to the pipeline
- Addition of LSModel S3 Wrapper class to create a flexible method for applying models of different types,
- `apply_model` is now a prototype function (via `useMethod(.)`) that applies models of varied types within the pipeline.
     - `apply_model.ranger` is now defined using the same behaviour as was in the pipeline before
     - `apply_model.LSModel` is defined.  It uses the apply function in the wrapper, allowing it to be used with any model or group of models.
 - Re-ran updated validation.

### CODE REVIEW

**Comment by krbundy** [comment]
Branch Conflicts were resolved by keeping the state of dev intact - all the conflicts were created by running an automated code formatter over one of the older scripts.  I left the state on dev intact, figuring it would create the least issues in the long run, in case there was some other change I could not see.
